src/main.js

```diff
@@ -98,29 +101,18 @@ var quotes = [
   "Each person must live their life as a model for others.",
   "A champion is defined not by their wins but by how they can recover when they fall."
 ];
-var savedPosters = [];
-var currentPoster;
+
 
 // event listeners go here 👇
+window.addEventListener('load', (event) => {
+  randomImage.src = images[getRandomIndex(images)];
+  randomTitle.innerText = titles[getRandomIndex(titles)];
+  randomQuote.innerText = quotes[getRandomIndex(quotes)];
+});
+
 
 // functions and event handlers go here 👇
 // (we've provided one for you to get you started)!
 function getRandomIndex(array) {
   return Math.floor(Math.random() * array.length);
 }
-
-// -----------------------------MAIN POSTER---------------------------------------
```

Comment:
@J-Dunny Hey! I understand you were were trying to create structure to our implementation file but after reviewing I don't think it's necessary. I'm happy to discuss.